config/locales/en/casework.yml

```diff
@@ -195,6 +197,7 @@ en:
       sent_back: red
       completed: green
       marked_as_ready: blue
+    download_file: "Download file <br> (%{content_type}, %{file_size})"
```

Comment:
Instead of `content_type` perhaps `file_type` or `file_extension` is more close to what we are doing/presenting? No blocker at all just a suggestion.

Reply:
Yeah `file_extension` is most accurate probably, will change